content/docs/rfcs/10/README.md

```diff
@@ -133,6 +133,12 @@ For an experimental privacy-preserving economic spam protection mechanism, see [
 
 See [23/WAKU2-TOPICS](/spec/23) for more information about recommended topic usage.
 
+# Transport Protocol Specification
+
+This section describes the specification for transport protocols supported by waku v2. While Waku strives to be transport agonistic, however any underlying transport layer MUST be a reliable transport.
+
+Waku client implementations SHOULD support the TCP transport, and MUST be enabled for both dialing and listening even if other transports are available if TCP is available.
```

Comment:
We try to use https://sembr.org/

Reply:
I'd add a sentence here saying that implementations running in certain restricted environments, such as browsers, may not be able to use TCP directly.

Reply:
```suggestion
Waku client implementations SHOULD support the TCP transport
Where TCP is supported it MUST be enabled for both dialing and listening, even if other transports are available.
```

Reply:
And agree with adding a comment on environments where TCP might not be appropriate.

Reply:
done in 3c212ea20df7ffb3ee009d1dce5b11fec4537a2c